### ISSUE TITLE
generating  secret key

### ISSUE THREAD

**Issue by martcpp** [github/codespaces-django#35]
there should be a python script to help user generate the django secret key for  any user

**Comment by snipher-marube** [comment]
I agree on this something I have been thinking about should be implemented but again we need others to learn so I think other users should atleast learn to generate the keys by themselves

**Comment by martcpp** [comment]
no issue i taught the aim is to make and easy setup for developer
